tests/integration-tests/src/test/resources/QpidRouterPeerTest-qpidr.conf

```diff
@@ -0,0 +1,51 @@
+ #
+ # Licensed to the Apache Software Foundation (ASF) under one or more
+ # contributor license agreements. See the NOTICE file distributed with
+ # this work for additional information regarding copyright ownership.
+ # The ASF licenses this file to You under the Apache License, Version 2.0
+ # (the "License"); you may not use this file except in compliance with
+ # the License. You may obtain a copy of the License at
+ #
+ #     http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+ #
+
+ router {
+     mode: standalone
+     id: INT.A
+ }
+#log {
+    #module: DEFAULT
+    #enable: trace+
+    #outputFile: /tmp/qdrouterd.log
+#}
+
+ # The broker connects into this port
+ listener {
+     saslMechanisms: ANONYMOUS
+     host: 0.0.0.0
+     role: route-container
+     linkCapacity: 1123
+     authenticatePeer: no
+     port: 24621
+ }
+
```

Comment:
Per the offline discussion yesterday, this listener isnt required and is somewhat misleading implying it has to be a route-container. The single normal (client) listener can be used.